Source/AFError.swift

```diff
@@ -548,6 +644,8 @@ extension AFError.ResponseSerializationFailureReason {
             return "Empty response could not be serialized to type: \(type). Use Empty as the expected type for such responses."
         case .decodingFailed(let error):
             return "Response could not be decoded because of error:\n\(error.localizedDescription)"
+        case .customSerializationFailed(let error):
+            return "Custom response could not be serialized because of error:\n\(error.localizedDescription)"
```

Comment:
This should be "Custom response serializer failed with error:"

Reply:
Done.